### DISCUSSION TITLE
Nested property in table header accessor?

### DISCUSSION THREAD

**Discussion by szmarci** [TanStack/table#2274] [General] [answered]
The API I'm using returning data in this format:
```
{
  id: "someid",
  pr.productnumber: 123455
}
```
Where some of the properties are in the dot notation format `/pr.productnumber/`.
However, when constructing headers for the table:
```
{
  Header: 'Product ID',
  accessor: 'pr.productnumber',
  minWidth: 100,
  width: 100,
  maxWidth: 200
},
```
The Product ID is not visible in the table. I'm guessing because of the nested property.
Is it possible to do this without mapping over all of my API calls, replacing nested props? (There's a lot...)

**Comment by BrianMitchL** [comment, marked as answer]
You can use a function for the assessor! Try:
```js
{
  Header: 'Product ID',
  accessor: (item) => item.pr.productnumber,
  minWidth: 100,
  width: 100,
  maxWidth: 200
},
```

**Reply by szmarci** [reply]
Thanks, it works!
But only like this:
`accessor: (item) => item["pr.productnumber"],`
